### ISSUE TITLE
Is it possible to make the emphasized  section in a sunburst graph to pop out and be slightly bigger?

### ISSUE THREAD

**Issue by wp-zg** [apache/echarts#16013]
### What problem does this feature solve?
This will help the emphasis option be more dynamic. The emphasized sections would pop up and be displayed slightly bigger compared to the others.

### What does the proposed API look like?
The idea I have in my would be something like this :
emphasis: {
    focus: ‘ancestor’,
    popUp: true,
    label: {
          rotate: 0,
    }
}

**Comment by Ovilia** [comment]
No, not for now. If you are interested in making a PR to enable it, we are willing to help you so. Please check out [wiki](https://github.com/apache/echarts/wiki) to learn more about how to contribute.

The new APIs should be similar to [those of pie charts](https://echarts.apache.org/en/option.html#series-pie.emphasis.scale). But please more care about [sunburst.emphasis.focus](https://echarts.apache.org/en/option.html#series-sunburst.emphasis.focus) since all emphasized items should be scaled.
